cmd/av/stack_branch.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"emperror.dev/errors"
+	"github.com/aviator-co/av/internal/git"
+	"github.com/aviator-co/av/internal/meta"
+	"github.com/aviator-co/av/internal/utils/cleanup"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var stackBranchFlags struct {
+	// The parent branch to base the new branch off.
+	// By default, this is the current branch.
+	Parent string
+}
+var stackBranchCmd = &cobra.Command{
+	Use:   "branch [flags] <branch-name>",
+	Short: "create a new stacked branch",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			_ = cmd.Usage()
+			return errors.New("exactly one branch name is required")
+		}
+
+		var cu cleanup.Cleanup
+		defer cu.Cleanup()
+
+		repo, err := getRepo()
+		if err != nil {
+			return err
+		}
+
+		name := args[0]
+
+		// validate operation preconditions
```

Comment:
Inlined what used to be `stacks.CreateBranch` here. I think the whole `stacks` package was a little bit of a premature abstraction.